src/test/java/collab/ModifyTest.java

```diff
@@ -27,60 +27,50 @@ public void modifyTest_returnEmployee() {
     @Test
     public void modifyTest_Name() {
       //"17112609","FB NTAWR","CL4","010-5645-6122","19860903","PRO"
-      
-      // name 변경 Test
       empDB.modifyItemByCondition("employeeNum", "17112609", "name", "TEST TEST");
 
-      assertTrue(empDB.searchItems("name","FB NTAWR").size()==0);
-      assertTrue(empDB.searchItems("name","TEST TEST").size()==1);
+      assertEquals(empDB.searchItems("name","FB NTAWR").size(),0);
+      assertTrue(empDB.searchItems("name","TEST TEST").get(0).getEmployeeNumber() == "17112609");
     }
     
     @Test
     public void modifyTest_Cl() {
       //"17112609","FB NTAWR","CL4","010-5645-6122","19860903","PRO"
-      
-      // cl 변경 Test
-      assertThrows(RuntimeException.class, () -> {
-        empDB.modifyItemByCondition("employeeNum", "17112609", "cl", "CL5");
-
-        assertTrue(empDB.searchItems("cl","CL4").size()==0);
-        assertTrue(empDB.searchItems("cl","CL5").size()==1);
-      });
+        empDB.modifyItemByCondition("employeeNum", "17112609", "cl", "CL1");
 
+        assertEquals(empDB.searchItems("cl","CL4").size(),0);
```

Comment:
assertEquals(기대값, 실제값)으로 쓰도록 정의되어 있네요.
실패했을 때 결과 표시도 Expected, Actual로 나오는 것으로 보아 이 형식에 맞추는 것이 좋을 것 같습니다.

```suggestion
        assertEquals(0, empDB.searchItems("cl","CL4").size());
```